.travis.yml

```diff
@@ -2,4 +2,3 @@ rvm:
   - 2.1.1
   - 2.1.0
   - 2.0.0
-  - 1.9.3
```

Comment:
do we have no customers using this? if we don't, fine - if we do, plz figure out how to notify them.